fix: review applications.md file

sr-remsha opened the pull request:
### Description of changes

Fixed errors and edited dscriptions

### Checklist

<!-- [Place an '[X]' (no spaces) in all applicable fields. Please remove unrelated fields.] -->

- [x] Title of the pull request follows [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

sr-remsha committed:
fix: Fix typo in editorUrl field description

ai-dial-actions commented:
<h1>Dependency Review</h1>
The following issues were found:<ul><li>✅ 0 vulnerable package(s)</li><li>✅ 0 package(s) with incompatible licenses</li><li>✅ 0 package(s) with invalid SPDX license definitions</li><li>⚠️ 23 package(s) with unknown licenses.</li><li>⚠️ 3 packages with OpenSSF Scorecard issues.</li></ul>
See the Details below.<h2>License Issues</h2>
<h4><em>settings.gradle</em></h4>
<table><tr><td>Package</td><td>Version</td><td>License</td><td>Issue Type</td></tr><tr><td>com.gradle:common-custom-user-data-gradle-plugin</td><td>2.1</td><td>Null</td><td>Unknown License</td></tr><tr><td>com.gradle:develocity-gradle-plugin</td><td>3.19.2</td><td>Null</td><td>Unknown License</td></tr><tr><td>org.apache.jclouds.api:atmos</td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td>org.apache.jclouds.api:filesystem</td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td>org.apache.jclouds.api:glacier</td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td>org.apache.jclouds.api:oauth</td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td>org.apache.jclouds.api:openstack-keystone</td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td>org.apache.jclouds.api:openstack-swift</td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td>org.apache.jclouds.api:rackspace-cloudfiles</td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td>org.apache.jclouds.api:rackspace-cloudidentity</td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td>org.apache.jclouds.api:s3</td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td>org.apache.jclouds.api:sts</td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td>org.apache.jclouds.common:googlecloud</td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td>org.apache.jclouds.provider:aws-s3</td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td>org.apache.jclouds.provider:azureblob</td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td>org.apache.jclouds.provider:b2</td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td>org.apache.jclouds.provider:google-cloud-storage</td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td>org.apache.jclouds.provider:rackspace-cloudfiles-uk</td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td>org.apache.jclouds.provider:rackspace-cloudfiles-us</td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td>org.glassfish.jaxb:txw2</td><td>4.0.5</td><td>Null</td><td>Unknown License</td></tr><tr><td><a href=https://github.com/jclouds/jclouds>org.apache.jclouds:jclouds-allblobstore</a></td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td><a href=https://github.com/jclouds/jclouds>org.apache.jclouds:jclouds-blobstore</a></td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td><a href=https://github.com/jclouds/jclouds>org.apache.jclouds:jclouds-core</a></td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr></table>
<h2>OpenSSF Scorecard</h2>
<details><summary>Scorecard details</summary>
<table><tr><th>Package</th><th>Version</th><th>Score</th><th>Details</th></tr>
<tr><td><a href=https://github.com/vastatrix/aopalliance> maven/aopalliance:aopalliance </a></td><td>1.0</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td><a href=https://github.com/google/guice> maven/com.google.inject.extensions:guice-assistedinject </a></td><td>7.0.0</td>
      <td>:green_circle: 5.8</td><td><details><summary>Details</summary><table><tr><th>Check</th><th>Score</th><th>Reason</th></tr><tr><td>Maintained</td><td>:warning: 0</td><td>0 commit(s) and 0 issue activity found in the last 90 days -- score normalized to 0</td></tr><tr><td>Dangerous-Workflow</td><td>:green_circle: 10</td><td>no dangerous workflow patterns detected</td></tr><tr><td>Packaging</td><td>:warning: -1</td><td>packaging workflow not detected</td></tr><tr><td>Code-Review</td><td>:warning: 1</td><td>Found 4/29 approved changesets -- score normalized to 1</td></tr><tr><td>Token-Permissions</td><td>:warning: 0</td><td>detected GitHub workflow tokens with excessive permissions</td></tr><tr><td>CII-Best-Practices</td><td>:warning: 0</td><td>no effort to earn an OpenSSF best practices badge detected</td></tr><tr><td>License</td><td>:green_circle: 10</td><td>license file detected</td></tr><tr><td>Binary-Artifacts</td><td>:green_circle: 10</td><td>no binaries found in the repo</td></tr><tr><td>Pinned-Dependencies</td><td>:green_circle: 10</td><td>all dependencies are pinned</td></tr><tr><td>Fuzzing</td><td>:green_circle: 10</td><td>project is fuzzed</td></tr><tr><td>Branch-Protection</td><td>:warning: -1</td><td>internal error: error during branchesHandler.setup: internal error: some github tokens can't read classic branch protection rules: https://github.com/ossf/scorecard-action/blob/main/docs/authentication/fine-grained-auth-token.md</td></tr><tr><td>Security-Policy</td><td>:green_circle: 10</td><td>security policy file detected</td></tr><tr><td>Signed-Releases</td><td>:warning: -1</td><td>no releases found</td></tr><tr><td>SAST</td><td>:warning: 0</td><td>SAST tool is not run on all commits -- score normalized to 0</td></tr><tr><td>Vulnerabilities</td><td>:green_circle: 8</td><td>2 existing vulnerabilities detected</td></tr></table></details></td></tr>
<tr><td><a href=https://github.com/google/guice> maven/com.google.inject:guice </a></td><td>7.0.0</td>
      <td>:green_circle: 5.8</td><td><details><summary>Details</summary><table><tr><th>Check</th><th>Score</th><th>Reason</th></tr><tr><td>Maintained</td><td>:warning: 0</td><td>0 commit(s) and 0 issue activity found in the last 90 days -- score normalized to 0</td></tr><tr><td>Dangerous-Workflow</td><td>:green_circle: 10</td><td>no dangerous workflow patterns detected</td></tr><tr><td>Packaging</td><td>:warning: -1</td><td>packaging workflow not detected</td></tr><tr><td>Code-Review</td><td>:warning: 1</td><td>Found 4/29 approved changesets -- score normalized to 1</td></tr><tr><td>Token-Permissions</td><td>:warning: 0</td><td>detected GitHub workflow tokens with excessive permissions</td></tr><tr><td>CII-Best-Practices</td><td>:warning: 0</td><td>no effort to earn an OpenSSF best practices badge detected</td></tr><tr><td>License</td><td>:green_circle: 10</td><td>license file detected</td></tr><tr><td>Binary-Artifacts</td><td>:green_circle: 10</td><td>no binaries found in the repo</td></tr><tr><td>Pinned-Dependencies</td><td>:green_circle: 10</td><td>all dependencies are pinned</td></tr><tr><td>Fuzzing</td><td>:green_circle: 10</td><td>project is fuzzed</td></tr><tr><td>Branch-Protection</td><td>:warning: -1</td><td>internal error: error during branchesHandler.setup: internal error: some github tokens can't read classic branch protection rules: https://github.com/ossf/scorecard-action/blob/main/docs/authentication/fine-grained-auth-token.md</td></tr><tr><td>Security-Policy</td><td>:green_circle: 10</td><td>security policy file detected</td></tr><tr><td>Signed-Releases</td><td>:warning: -1</td><td>no releases found</td></tr><tr><td>SAST</td><td>:warning: 0</td><td>SAST tool is not run on all commits -- score normalized to 0</td></tr><tr><td>Vulnerabilities</td><td>:green_circle: 8</td><td>2 existing vulnerabilities detected</td></tr></table></details></td></tr>
<tr><td> maven/com.gradle:common-custom-user-data-gradle-plugin </td><td>2.1</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td> maven/com.gradle:develocity-gradle-plugin </td><td>3.19.2</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td><a href=https://github.com/eclipse-ee4j/jaxb-istack-commons> maven/com.sun.istack:istack-commons-runtime </a></td><td>4.1.2</td>
      <td>:green_circle: 5.5</td><td><details><summary>Details</summary><table><tr><th>Check</th><th>Score</th><th>Reason</th></tr><tr><td>Maintained</td><td>:warning: 0</td><td>0 commit(s) and 0 issue activity found in the last 90 days -- score normalized to 0</td></tr><tr><td>Code-Review</td><td>:green_circle: 10</td><td>all changesets reviewed</td></tr><tr><td>Packaging</td><td>:warning: -1</td><td>packaging workflow not detected</td></tr><tr><td>Dangerous-Workflow</td><td>:green_circle: 10</td><td>no dangerous workflow patterns detected</td></tr><tr><td>Pinned-Dependencies</td><td>:warning: 0</td><td>dependency not pinned by hash detected -- score normalized to 0</td></tr><tr><td>Token-Permissions</td><td>:warning: 0</td><td>detected GitHub workflow tokens with excessive permissions</td></tr><tr><td>Binary-Artifacts</td><td>:green_circle: 10</td><td>no binaries found in the repo</td></tr><tr><td>CII-Best-Practices</td><td>:warning: 0</td><td>no effort to earn an OpenSSF best practices badge detected</td></tr><tr><td>Fuzzing</td><td>:warning: 0</td><td>project is not fuzzed</td></tr><tr><td>License</td><td>:green_circle: 10</td><td>license file detected</td></tr><tr><td>Signed-Releases</td><td>:warning: -1</td><td>no releases found</td></tr><tr><td>Branch-Protection</td><td>:warning: -1</td><td>internal error: error during branchesHandler.setup: internal error: some github tokens can't read classic branch protection rules: https://github.com/ossf/scorecard-action/blob/main/docs/authentication/fine-grained-auth-token.md</td></tr><tr><td>Security-Policy</td><td>:green_circle: 10</td><td>security policy file detected</td></tr><tr><td>SAST</td><td>:warning: 0</td><td>SAST tool is not run on all commits -- score normalized to 0</td></tr><tr><td>Vulnerabilities</td><td>:green_circle: 10</td><td>0 existing vulnerabilities detected</td></tr></table></details></td></tr>
<tr><td><a href=https://github.com/jakartaee/jaf-api> maven/jakarta.activation:jakarta.activation-api </a></td><td>2.1.3</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td><a href=https://github.com/jakartaee/common-annotations-api> maven/jakarta.annotation:jakarta.annotation-api </a></td><td>2.0.0</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td><a href=https://github.com/jakartaee/inject> maven/jakarta.inject:jakarta.inject-api </a></td><td>2.0.1</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td><a href=https://github.com/jakartaee/rest> maven/jakarta.ws.rs:jakarta.ws.rs-api </a></td><td>3.0.0</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td><a href=https://github.com/jakartaee/jaxb-api> maven/jakarta.xml.bind:jakarta.xml.bind-api </a></td><td>4.0.2</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td> maven/org.apache.jclouds.api:atmos </td><td>2.7.3</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td> maven/org.apache.jclouds.api:filesystem </td><td>2.7.3</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td> maven/org.apache.jclouds.api:glacier </td><td>2.7.3</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td> maven/org.apache.jclouds.api:oauth </td><td>2.7.3</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td> maven/org.apache.jclouds.api:openstack-keystone </td><td>2.7.3</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td> maven/org.apache.jclouds.api:openstack-swift </td><td>2.7.3</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td> maven/org.apache.jclouds.api:rackspace-cloudfiles </td><td>2.7.3</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td> maven/org.apache.jclouds.api:rackspace-cloudidentity </td><td>2.7.3</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td> maven/org.apache.jclouds.api:s3 </td><td>2.7.3</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td> maven/org.apache.jclouds.api:sts </td><td>2.7.3</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td> maven/org.apache.jclouds.common:googlecloud </td><td>2.7.3</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td> maven/org.apache.jclouds.provider:aws-s3 </td><td>2.7.3</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td> maven/org.apache.jclouds.provider:azureblob </td><td>2.7.3</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td> maven/org.apache.jclouds.provider:b2 </td><td>2.7.3</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td> maven/org.apache.jclouds.provider:google-cloud-storage </td><td>2.7.3</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td> maven/org.apache.jclouds.provider:rackspace-cloudfiles-uk </td><td>2.7.3</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td> maven/org.apache.jclouds.provider:rackspace-cloudfiles-us </td><td>2.7.3</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td><a href=https://github.com/jclouds/jclouds> maven/org.apache.jclouds:jclouds-allblobstore </a></td><td>2.7.3</td>
      <td>:warning: 2.5</td><td><details><summary>Details</summary><table><tr><th>Check</th><th>Score</th><th>Reason</th></tr><tr><td>Packaging</td><td>:warning: -1</td><td>packaging workflow not detected</td></tr><tr><td>Code-Review</td><td>:warning: 0</td><td>Found 2/30 approved changesets -- score normalized to 0</td></tr><tr><td>Dangerous-Workflow</td><td>:warning: -1</td><td>no workflows found</td></tr><tr><td>Maintained</td><td>:warning: 0</td><td>project is archived</td></tr><tr><td>Token-Permissions</td><td>:warning: -1</td><td>No tokens found</td></tr><tr><td>CII-Best-Practices</td><td>:warning: 0</td><td>no effort to earn an OpenSSF best practices badge detected</td></tr><tr><td>Security-Policy</td><td>:warning: 0</td><td>security policy file not detected</td></tr><tr><td>License</td><td>:warning: 0</td><td>license file not detected</td></tr><tr><td>Binary-Artifacts</td><td>:green_circle: 10</td><td>no binaries found in the repo</td></tr><tr><td>Signed-Releases</td><td>:warning: -1</td><td>no releases found</td></tr><tr><td>Pinned-Dependencies</td><td>:warning: -1</td><td>no dependencies found</td></tr><tr><td>Fuzzing</td><td>:warning: 0</td><td>project is not fuzzed</td></tr><tr><td>Branch-Protection</td><td>:warning: 0</td><td>branch protection not enabled on development/release branches</td></tr><tr><td>SAST</td><td>:warning: 0</td><td>SAST tool is not run on all commits -- score normalized to 0</td></tr><tr><td>Vulnerabilities</td><td>:green_circle: 9</td><td>1 existing vulnerabilities detected</td></tr></table></details></td></tr>
<tr><td><a href=https://github.com/jclouds/jclouds> maven/org.apache.jclouds:jclouds-blobstore </a></td><td>2.7.3</td>
      <td>:warning: 2.5</td><td><details><summary>Details</summary><table><tr><th>Check</th><th>Score</th><th>Reason</th></tr><tr><td>Packaging</td><td>:warning: -1</td><td>packaging workflow not detected</td></tr><tr><td>Code-Review</td><td>:warning: 0</td><td>Found 2/30 approved changesets -- score normalized to 0</td></tr><tr><td>Dangerous-Workflow</td><td>:warning: -1</td><td>no workflows found</td></tr><tr><td>Maintained</td><td>:warning: 0</td><td>project is archived</td></tr><tr><td>Token-Permissions</td><td>:warning: -1</td><td>No tokens found</td></tr><tr><td>CII-Best-Practices</td><td>:warning: 0</td><td>no effort to earn an OpenSSF best practices badge detected</td></tr><tr><td>Security-Policy</td><td>:warning: 0</td><td>security policy file not detected</td></tr><tr><td>License</td><td>:warning: 0</td><td>license file not detected</td></tr><tr><td>Binary-Artifacts</td><td>:green_circle: 10</td><td>no binaries found in the repo</td></tr><tr><td>Signed-Releases</td><td>:warning: -1</td><td>no releases found</td></tr><tr><td>Pinned-Dependencies</td><td>:warning: -1</td><td>no dependencies found</td></tr><tr><td>Fuzzing</td><td>:warning: 0</td><td>project is not fuzzed</td></tr><tr><td>Branch-Protection</td><td>:warning: 0</td><td>branch protection not enabled on development/release branches</td></tr><tr><td>SAST</td><td>:warning: 0</td><td>SAST tool is not run on all commits -- score normalized to 0</td></tr><tr><td>Vulnerabilities</td><td>:green_circle: 9</td><td>1 existing vulnerabilities detected</td></tr></table></details></td></tr>
<tr><td><a href=https://github.com/jclouds/jclouds> maven/org.apache.jclouds:jclouds-core </a></td><td>2.7.3</td>
      <td>:warning: 2.5</td><td><details><summary>Details</summary><table><tr><th>Check</th><th>Score</th><th>Reason</th></tr><tr><td>Packaging</td><td>:warning: -1</td><td>packaging workflow not detected</td></tr><tr><td>Code-Review</td><td>:warning: 0</td><td>Found 2/30 approved changesets -- score normalized to 0</td></tr><tr><td>Dangerous-Workflow</td><td>:warning: -1</td><td>no workflows found</td></tr><tr><td>Maintained</td><td>:warning: 0</td><td>project is archived</td></tr><tr><td>Token-Permissions</td><td>:warning: -1</td><td>No tokens found</td></tr><tr><td>CII-Best-Practices</td><td>:warning: 0</td><td>no effort to earn an OpenSSF best practices badge detected</td></tr><tr><td>Security-Policy</td><td>:warning: 0</td><td>security policy file not detected</td></tr><tr><td>License</td><td>:warning: 0</td><td>license file not detected</td></tr><tr><td>Binary-Artifacts</td><td>:green_circle: 10</td><td>no binaries found in the repo</td></tr><tr><td>Signed-Releases</td><td>:warning: -1</td><td>no releases found</td></tr><tr><td>Pinned-Dependencies</td><td>:warning: -1</td><td>no dependencies found</td></tr><tr><td>Fuzzing</td><td>:warning: 0</td><td>project is not fuzzed</td></tr><tr><td>Branch-Protection</td><td>:warning: 0</td><td>branch protection not enabled on development/release branches</td></tr><tr><td>SAST</td><td>:warning: 0</td><td>SAST tool is not run on all commits -- score normalized to 0</td></tr><tr><td>Vulnerabilities</td><td>:green_circle: 9</td><td>1 existing vulnerabilities detected</td></tr></table></details></td></tr>
<tr><td> maven/org.checkerframework:checker-qual </td><td>3.43.0</td>
      <td>:green_circle: 3.5</td><td><details><summary>Details</summary><table><tr><th>Check</th><th>Score</th><th>Reason</th></tr><tr><td>Packaging</td><td>:warning: -1</td><td>packaging workflow not detected</td></tr><tr><td>Code-Review</td><td>:green_circle: 3</td><td>Found 10/27 approved changesets -- score normalized to 3</td></tr><tr><td>Maintained</td><td>:green_circle: 10</td><td>30 commit(s) and 6 issue activity found in the last 90 days -- score normalized to 10</td></tr><tr><td>Dangerous-Workflow</td><td>:warning: -1</td><td>no workflows found</td></tr><tr><td>Token-Permissions</td><td>:warning: -1</td><td>No tokens found</td></tr><tr><td>CII-Best-Practices</td><td>:warning: 0</td><td>no effort to earn an OpenSSF best practices badge detected</td></tr><tr><td>Security-Policy</td><td>:warning: 0</td><td>security policy file not detected</td></tr><tr><td>License</td><td>:green_circle: 9</td><td>license file detected</td></tr><tr><td>Branch-Protection</td><td>:warning: -1</td><td>internal error: error during branchesHandler.setup: internal error: some github tokens can't read classic branch protection rules: https://github.com/ossf/scorecard-action/blob/main/docs/authentication/fine-grained-auth-token.md</td></tr><tr><td>Signed-Releases</td><td>:warning: 0</td><td>Project has not signed or included provenance with any releases.</td></tr><tr><td>Binary-Artifacts</td><td>:warning: -1</td><td>internal error: failure checking for Gradle wrapper validating Action: failure listing workflow runs: internal error: ListWorkflowRunsByFileName: GET https://api.github.com/repos/typetools/checker-framework/actions/workflows/gradle-wrapper-validation.yml-DISABLED/runs?status=success: 404 Not Found []</td></tr><tr><td>Fuzzing</td><td>:warning: 0</td><td>project is not fuzzed</td></tr><tr><td>Vulnerabilities</td><td>:green_circle: 10</td><td>0 existing vulnerabilities detected</td></tr><tr><td>SAST</td><td>:warning: 0</td><td>SAST tool is not run on all commits -- score normalized to 0</td></tr><tr><td>Pinned-Dependencies</td><td>:warning: 0</td><td>dependency not pinned by hash detected -- score normalized to 0</td></tr></table></details></td></tr>
<tr><td><a href=https://github.com/eclipse-ee4j/angus-activation> maven/org.eclipse.angus:angus-activation </a></td><td>2.0.2</td>
      <td>:green_circle: 3.7</td><td><details><summary>Details</summary><table><tr><th>Check</th><th>Score</th><th>Reason</th></tr><tr><td>Dangerous-Workflow</td><td>:green_circle: 10</td><td>no dangerous workflow patterns detected</td></tr><tr><td>Packaging</td><td>:warning: -1</td><td>packaging workflow not detected</td></tr><tr><td>Maintained</td><td>:green_circle: 4</td><td>5 commit(s) and 0 issue activity found in the last 90 days -- score normalized to 4</td></tr><tr><td>Binary-Artifacts</td><td>:green_circle: 10</td><td>no binaries found in the repo</td></tr><tr><td>Code-Review</td><td>:warning: 2</td><td>Found 5/22 approved changesets -- score normalized to 2</td></tr><tr><td>Token-Permissions</td><td>:warning: 0</td><td>detected GitHub workflow tokens with excessive permissions</td></tr><tr><td>Pinned-Dependencies</td><td>:warning: 0</td><td>dependency not pinned by hash detected -- score normalized to 0</td></tr><tr><td>CII-Best-Practices</td><td>:warning: 0</td><td>no effort to earn an OpenSSF best practices badge detected</td></tr><tr><td>Fuzzing</td><td>:warning: 0</td><td>project is not fuzzed</td></tr><tr><td>License</td><td>:green_circle: 10</td><td>license file detected</td></tr><tr><td>Branch-Protection</td><td>:warning: 0</td><td>branch protection not enabled on development/release branches</td></tr><tr><td>Signed-Releases</td><td>:warning: -1</td><td>no releases found</td></tr><tr><td>Security-Policy</td><td>:green_circle: 10</td><td>security policy file detected</td></tr><tr><td>SAST</td><td>:warning: 0</td><td>SAST tool is not run on all commits -- score normalized to 0</td></tr><tr><td>Vulnerabilities</td><td>:warning: 0</td><td>22 existing vulnerabilities detected</td></tr></table></details></td></tr>
<tr><td> maven/org.glassfish.jaxb:jaxb-core </td><td>4.0.5</td>
      <td>:green_circle: 6.6</td><td><details><summary>Details</summary><table><tr><th>Check</th><th>Score</th><th>Reason</th></tr><tr><td>Code-Review</td><td>:green_circle: 6</td><td>Found 17/25 approved changesets -- score normalized to 6</td></tr><tr><td>Maintained</td><td>:green_circle: 10</td><td>13 commit(s) and 2 issue activity found in the last 90 days -- score normalized to 10</td></tr><tr><td>Packaging</td><td>:warning: -1</td><td>packaging workflow not detected</td></tr><tr><td>Dangerous-Workflow</td><td>:green_circle: 10</td><td>no dangerous workflow patterns detected</td></tr><tr><td>Token-Permissions</td><td>:warning: 0</td><td>detected GitHub workflow tokens with excessive permissions</td></tr><tr><td>CII-Best-Practices</td><td>:warning: 0</td><td>no effort to earn an OpenSSF best practices badge detected</td></tr><tr><td>License</td><td>:green_circle: 10</td><td>license file detected</td></tr><tr><td>Binary-Artifacts</td><td>:green_circle: 10</td><td>no binaries found in the repo</td></tr><tr><td>Pinned-Dependencies</td><td>:warning: 0</td><td>dependency not pinned by hash detected -- score normalized to 0</td></tr><tr><td>Branch-Protection</td><td>:warning: 0</td><td>branch protection not enabled on development/release branches</td></tr><tr><td>Security-Policy</td><td>:green_circle: 10</td><td>security policy file detected</td></tr><tr><td>Fuzzing</td><td>:green_circle: 10</td><td>project is fuzzed</td></tr><tr><td>Signed-Releases</td><td>:warning: -1</td><td>no releases found</td></tr><tr><td>SAST</td><td>:green_circle: 7</td><td>SAST tool detected but not run on all commits</td></tr><tr><td>Vulnerabilities</td><td>:green_circle: 10</td><td>0 existing vulnerabilities detected</td></tr></table></details></td></tr>
<tr><td> maven/org.glassfish.jaxb:jaxb-runtime </td><td>4.0.5</td>
      <td>:green_circle: 6.6</td><td><details><summary>Details</summary><table><tr><th>Check</th><th>Score</th><th>Reason</th></tr><tr><td>Code-Review</td><td>:green_circle: 6</td><td>Found 17/25 approved changesets -- score normalized to 6</td></tr><tr><td>Maintained</td><td>:green_circle: 10</td><td>13 commit(s) and 2 issue activity found in the last 90 days -- score normalized to 10</td></tr><tr><td>Packaging</td><td>:warning: -1</td><td>packaging workflow not detected</td></tr><tr><td>Dangerous-Workflow</td><td>:green_circle: 10</td><td>no dangerous workflow patterns detected</td></tr><tr><td>Token-Permissions</td><td>:warning: 0</td><td>detected GitHub workflow tokens with excessive permissions</td></tr><tr><td>CII-Best-Practices</td><td>:warning: 0</td><td>no effort to earn an OpenSSF best practices badge detected</td></tr><tr><td>License</td><td>:green_circle: 10</td><td>license file detected</td></tr><tr><td>Binary-Artifacts</td><td>:green_circle: 10</td><td>no binaries found in the repo</td></tr><tr><td>Pinned-Dependencies</td><td>:warning: 0</td><td>dependency not pinned by hash detected -- score normalized to 0</td></tr><tr><td>Branch-Protection</td><td>:warning: 0</td><td>branch protection not enabled on development/release branches</td></tr><tr><td>Security-Policy</td><td>:green_circle: 10</td><td>security policy file detected</td></tr><tr><td>Fuzzing</td><td>:green_circle: 10</td><td>project is fuzzed</td></tr><tr><td>Signed-Releases</td><td>:warning: -1</td><td>no releases found</td></tr><tr><td>SAST</td><td>:green_circle: 7</td><td>SAST tool detected but not run on all commits</td></tr><tr><td>Vulnerabilities</td><td>:green_circle: 10</td><td>0 existing vulnerabilities detected</td></tr></table></details></td></tr>
<tr><td> maven/org.glassfish.jaxb:txw2 </td><td>4.0.5</td>
      <td>:green_circle: 6.6</td><td><details><summary>Details</summary><table><tr><th>Check</th><th>Score</th><th>Reason</th></tr><tr><td>Code-Review</td><td>:green_circle: 6</td><td>Found 17/25 approved changesets -- score normalized to 6</td></tr><tr><td>Maintained</td><td>:green_circle: 10</td><td>13 commit(s) and 2 issue activity found in the last 90 days -- score normalized to 10</td></tr><tr><td>Packaging</td><td>:warning: -1</td><td>packaging workflow not detected</td></tr><tr><td>Dangerous-Workflow</td><td>:green_circle: 10</td><td>no dangerous workflow patterns detected</td></tr><tr><td>Token-Permissions</td><td>:warning: 0</td><td>detected GitHub workflow tokens with excessive permissions</td></tr><tr><td>CII-Best-Practices</td><td>:warning: 0</td><td>no effort to earn an OpenSSF best practices badge detected</td></tr><tr><td>License</td><td>:green_circle: 10</td><td>license file detected</td></tr><tr><td>Binary-Artifacts</td><td>:green_circle: 10</td><td>no binaries found in the repo</td></tr><tr><td>Pinned-Dependencies</td><td>:warning: 0</td><td>dependency not pinned by hash detected -- score normalized to 0</td></tr><tr><td>Branch-Protection</td><td>:warning: 0</td><td>branch protection not enabled on development/release branches</td></tr><tr><td>Security-Policy</td><td>:green_circle: 10</td><td>security policy file detected</td></tr><tr><td>Fuzzing</td><td>:green_circle: 10</td><td>project is fuzzed</td></tr><tr><td>Signed-Releases</td><td>:warning: -1</td><td>no releases found</td></tr><tr><td>SAST</td><td>:green_circle: 7</td><td>SAST tool detected but not run on all commits</td></tr><tr><td>Vulnerabilities</td><td>:green_circle: 10</td><td>0 existing vulnerabilities detected</td></tr></table></details></td></tr>
</table></details><h2>Scanned Files</h2>
<ul><li>settings.gradle</li></ul>

sr-remsha committed:
Fix grammar and clarity in applications documentation

Corrected grammar and improved clarity in the documentation regarding dynamic settings for applications.

ai-dial-actions commented:
<h1>Dependency Review</h1>
The following issues were found:<ul><li>✅ 0 vulnerable package(s)</li><li>✅ 0 package(s) with incompatible licenses</li><li>✅ 0 package(s) with invalid SPDX license definitions</li><li>⚠️ 23 package(s) with unknown licenses.</li><li>⚠️ 3 packages with OpenSSF Scorecard issues.</li></ul>
See the Details below.<h2>License Issues</h2>
<h4><em>settings.gradle</em></h4>
<table><tr><td>Package</td><td>Version</td><td>License</td><td>Issue Type</td></tr><tr><td>com.gradle:common-custom-user-data-gradle-plugin</td><td>2.1</td><td>Null</td><td>Unknown License</td></tr><tr><td>com.gradle:develocity-gradle-plugin</td><td>3.19.2</td><td>Null</td><td>Unknown License</td></tr><tr><td>org.apache.jclouds.api:atmos</td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td>org.apache.jclouds.api:filesystem</td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td>org.apache.jclouds.api:glacier</td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td>org.apache.jclouds.api:oauth</td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td>org.apache.jclouds.api:openstack-keystone</td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td>org.apache.jclouds.api:openstack-swift</td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td>org.apache.jclouds.api:rackspace-cloudfiles</td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td>org.apache.jclouds.api:rackspace-cloudidentity</td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td>org.apache.jclouds.api:s3</td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td>org.apache.jclouds.api:sts</td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td>org.apache.jclouds.common:googlecloud</td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td>org.apache.jclouds.provider:aws-s3</td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td>org.apache.jclouds.provider:azureblob</td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td>org.apache.jclouds.provider:b2</td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td>org.apache.jclouds.provider:google-cloud-storage</td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td>org.apache.jclouds.provider:rackspace-cloudfiles-uk</td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td>org.apache.jclouds.provider:rackspace-cloudfiles-us</td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td>org.glassfish.jaxb:txw2</td><td>4.0.5</td><td>Null</td><td>Unknown License</td></tr><tr><td><a href=https://github.com/jclouds/jclouds>org.apache.jclouds:jclouds-allblobstore</a></td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td><a href=https://github.com/jclouds/jclouds>org.apache.jclouds:jclouds-blobstore</a></td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr><tr><td><a href=https://github.com/jclouds/jclouds>org.apache.jclouds:jclouds-core</a></td><td>2.7.3</td><td>Null</td><td>Unknown License</td></tr></table>
<h2>OpenSSF Scorecard</h2>
<details><summary>Scorecard details</summary>
<table><tr><th>Package</th><th>Version</th><th>Score</th><th>Details</th></tr>
<tr><td><a href=https://github.com/vastatrix/aopalliance> maven/aopalliance:aopalliance </a></td><td>1.0</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td><a href=https://github.com/google/guice> maven/com.google.inject.extensions:guice-assistedinject </a></td><td>7.0.0</td>
      <td>:green_circle: 5.8</td><td><details><summary>Details</summary><table><tr><th>Check</th><th>Score</th><th>Reason</th></tr><tr><td>Code-Review</td><td>:warning: 1</td><td>Found 4/29 approved changesets -- score normalized to 1</td></tr><tr><td>Dangerous-Workflow</td><td>:green_circle: 10</td><td>no dangerous workflow patterns detected</td></tr><tr><td>Token-Permissions</td><td>:warning: 0</td><td>detected GitHub workflow tokens with excessive permissions</td></tr><tr><td>Packaging</td><td>:warning: -1</td><td>packaging workflow not detected</td></tr><tr><td>Maintained</td><td>:warning: 0</td><td>0 commit(s) and 0 issue activity found in the last 90 days -- score normalized to 0</td></tr><tr><td>CII-Best-Practices</td><td>:warning: 0</td><td>no effort to earn an OpenSSF best practices badge detected</td></tr><tr><td>Pinned-Dependencies</td><td>:green_circle: 10</td><td>all dependencies are pinned</td></tr><tr><td>Binary-Artifacts</td><td>:green_circle: 10</td><td>no binaries found in the repo</td></tr><tr><td>License</td><td>:green_circle: 10</td><td>license file detected</td></tr><tr><td>Fuzzing</td><td>:green_circle: 10</td><td>project is fuzzed</td></tr><tr><td>Signed-Releases</td><td>:warning: -1</td><td>no releases found</td></tr><tr><td>Branch-Protection</td><td>:warning: -1</td><td>internal error: error during branchesHandler.setup: internal error: some github tokens can't read classic branch protection rules: https://github.com/ossf/scorecard-action/blob/main/docs/authentication/fine-grained-auth-token.md</td></tr><tr><td>Security-Policy</td><td>:green_circle: 10</td><td>security policy file detected</td></tr><tr><td>SAST</td><td>:warning: 0</td><td>SAST tool is not run on all commits -- score normalized to 0</td></tr><tr><td>Vulnerabilities</td><td>:green_circle: 8</td><td>2 existing vulnerabilities detected</td></tr></table></details></td></tr>
<tr><td><a href=https://github.com/google/guice> maven/com.google.inject:guice </a></td><td>7.0.0</td>
      <td>:green_circle: 5.8</td><td><details><summary>Details</summary><table><tr><th>Check</th><th>Score</th><th>Reason</th></tr><tr><td>Code-Review</td><td>:warning: 1</td><td>Found 4/29 approved changesets -- score normalized to 1</td></tr><tr><td>Dangerous-Workflow</td><td>:green_circle: 10</td><td>no dangerous workflow patterns detected</td></tr><tr><td>Token-Permissions</td><td>:warning: 0</td><td>detected GitHub workflow tokens with excessive permissions</td></tr><tr><td>Packaging</td><td>:warning: -1</td><td>packaging workflow not detected</td></tr><tr><td>Maintained</td><td>:warning: 0</td><td>0 commit(s) and 0 issue activity found in the last 90 days -- score normalized to 0</td></tr><tr><td>CII-Best-Practices</td><td>:warning: 0</td><td>no effort to earn an OpenSSF best practices badge detected</td></tr><tr><td>Pinned-Dependencies</td><td>:green_circle: 10</td><td>all dependencies are pinned</td></tr><tr><td>Binary-Artifacts</td><td>:green_circle: 10</td><td>no binaries found in the repo</td></tr><tr><td>License</td><td>:green_circle: 10</td><td>license file detected</td></tr><tr><td>Fuzzing</td><td>:green_circle: 10</td><td>project is fuzzed</td></tr><tr><td>Signed-Releases</td><td>:warning: -1</td><td>no releases found</td></tr><tr><td>Branch-Protection</td><td>:warning: -1</td><td>internal error: error during branchesHandler.setup: internal error: some github tokens can't read classic branch protection rules: https://github.com/ossf/scorecard-action/blob/main/docs/authentication/fine-grained-auth-token.md</td></tr><tr><td>Security-Policy</td><td>:green_circle: 10</td><td>security policy file detected</td></tr><tr><td>SAST</td><td>:warning: 0</td><td>SAST tool is not run on all commits -- score normalized to 0</td></tr><tr><td>Vulnerabilities</td><td>:green_circle: 8</td><td>2 existing vulnerabilities detected</td></tr></table></details></td></tr>
<tr><td> maven/com.gradle:common-custom-user-data-gradle-plugin </td><td>2.1</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td> maven/com.gradle:develocity-gradle-plugin </td><td>3.19.2</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td><a href=https://github.com/eclipse-ee4j/jaxb-istack-commons> maven/com.sun.istack:istack-commons-runtime </a></td><td>4.1.2</td>
      <td>:green_circle: 5.5</td><td><details><summary>Details</summary><table><tr><th>Check</th><th>Score</th><th>Reason</th></tr><tr><td>Maintained</td><td>:warning: 0</td><td>0 commit(s) and 0 issue activity found in the last 90 days -- score normalized to 0</td></tr><tr><td>Code-Review</td><td>:green_circle: 10</td><td>all changesets reviewed</td></tr><tr><td>Packaging</td><td>:warning: -1</td><td>packaging workflow not detected</td></tr><tr><td>Dangerous-Workflow</td><td>:green_circle: 10</td><td>no dangerous workflow patterns detected</td></tr><tr><td>Pinned-Dependencies</td><td>:warning: 0</td><td>dependency not pinned by hash detected -- score normalized to 0</td></tr><tr><td>Token-Permissions</td><td>:warning: 0</td><td>detected GitHub workflow tokens with excessive permissions</td></tr><tr><td>Binary-Artifacts</td><td>:green_circle: 10</td><td>no binaries found in the repo</td></tr><tr><td>CII-Best-Practices</td><td>:warning: 0</td><td>no effort to earn an OpenSSF best practices badge detected</td></tr><tr><td>Fuzzing</td><td>:warning: 0</td><td>project is not fuzzed</td></tr><tr><td>License</td><td>:green_circle: 10</td><td>license file detected</td></tr><tr><td>Signed-Releases</td><td>:warning: -1</td><td>no releases found</td></tr><tr><td>Branch-Protection</td><td>:warning: -1</td><td>internal error: error during branchesHandler.setup: internal error: some github tokens can't read classic branch protection rules: https://github.com/ossf/scorecard-action/blob/main/docs/authentication/fine-grained-auth-token.md</td></tr><tr><td>Security-Policy</td><td>:green_circle: 10</td><td>security policy file detected</td></tr><tr><td>SAST</td><td>:warning: 0</td><td>SAST tool is not run on all commits -- score normalized to 0</td></tr><tr><td>Vulnerabilities</td><td>:green_circle: 10</td><td>0 existing vulnerabilities detected</td></tr></table></details></td></tr>
<tr><td><a href=https://github.com/jakartaee/jaf-api> maven/jakarta.activation:jakarta.activation-api </a></td><td>2.1.3</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td><a href=https://github.com/jakartaee/common-annotations-api> maven/jakarta.annotation:jakarta.annotation-api </a></td><td>2.0.0</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td><a href=https://github.com/jakartaee/inject> maven/jakarta.inject:jakarta.inject-api </a></td><td>2.0.1</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td><a href=https://github.com/jakartaee/rest> maven/jakarta.ws.rs:jakarta.ws.rs-api </a></td><td>3.0.0</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td><a href=https://github.com/jakartaee/jaxb-api> maven/jakarta.xml.bind:jakarta.xml.bind-api </a></td><td>4.0.2</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td> maven/org.apache.jclouds.api:atmos </td><td>2.7.3</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td> maven/org.apache.jclouds.api:filesystem </td><td>2.7.3</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td> maven/org.apache.jclouds.api:glacier </td><td>2.7.3</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td> maven/org.apache.jclouds.api:oauth </td><td>2.7.3</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td> maven/org.apache.jclouds.api:openstack-keystone </td><td>2.7.3</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td> maven/org.apache.jclouds.api:openstack-swift </td><td>2.7.3</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td> maven/org.apache.jclouds.api:rackspace-cloudfiles </td><td>2.7.3</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td> maven/org.apache.jclouds.api:rackspace-cloudidentity </td><td>2.7.3</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td> maven/org.apache.jclouds.api:s3 </td><td>2.7.3</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td> maven/org.apache.jclouds.api:sts </td><td>2.7.3</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td> maven/org.apache.jclouds.common:googlecloud </td><td>2.7.3</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td> maven/org.apache.jclouds.provider:aws-s3 </td><td>2.7.3</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td> maven/org.apache.jclouds.provider:azureblob </td><td>2.7.3</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td> maven/org.apache.jclouds.provider:b2 </td><td>2.7.3</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td> maven/org.apache.jclouds.provider:google-cloud-storage </td><td>2.7.3</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td> maven/org.apache.jclouds.provider:rackspace-cloudfiles-uk </td><td>2.7.3</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td> maven/org.apache.jclouds.provider:rackspace-cloudfiles-us </td><td>2.7.3</td>
      <td> Unknown</td><td>Unknown</td></tr>
<tr><td><a href=https://github.com/jclouds/jclouds> maven/org.apache.jclouds:jclouds-allblobstore </a></td><td>2.7.3</td>
      <td>:warning: 2.5</td><td><details><summary>Details</summary><table><tr><th>Check</th><th>Score</th><th>Reason</th></tr><tr><td>Packaging</td><td>:warning: -1</td><td>packaging workflow not detected</td></tr><tr><td>Code-Review</td><td>:warning: 0</td><td>Found 2/30 approved changesets -- score normalized to 0</td></tr><tr><td>Dangerous-Workflow</td><td>:warning: -1</td><td>no workflows found</td></tr><tr><td>Maintained</td><td>:warning: 0</td><td>project is archived</td></tr><tr><td>Token-Permissions</td><td>:warning: -1</td><td>No tokens found</td></tr><tr><td>CII-Best-Practices</td><td>:warning: 0</td><td>no effort to earn an OpenSSF best practices badge detected</td></tr><tr><td>Security-Policy</td><td>:warning: 0</td><td>security policy file not detected</td></tr><tr><td>License</td><td>:warning: 0</td><td>license file not detected</td></tr><tr><td>Binary-Artifacts</td><td>:green_circle: 10</td><td>no binaries found in the repo</td></tr><tr><td>Signed-Releases</td><td>:warning: -1</td><td>no releases found</td></tr><tr><td>Pinned-Dependencies</td><td>:warning: -1</td><td>no dependencies found</td></tr><tr><td>Fuzzing</td><td>:warning: 0</td><td>project is not fuzzed</td></tr><tr><td>Branch-Protection</td><td>:warning: 0</td><td>branch protection not enabled on development/release branches</td></tr><tr><td>SAST</td><td>:warning: 0</td><td>SAST tool is not run on all commits -- score normalized to 0</td></tr><tr><td>Vulnerabilities</td><td>:green_circle: 9</td><td>1 existing vulnerabilities detected</td></tr></table></details></td></tr>
<tr><td><a href=https://github.com/jclouds/jclouds> maven/org.apache.jclouds:jclouds-blobstore </a></td><td>2.7.3</td>
      <td>:warning: 2.5</td><td><details><summary>Details</summary><table><tr><th>Check</th><th>Score</th><th>Reason</th></tr><tr><td>Packaging</td><td>:warning: -1</td><td>packaging workflow not detected</td></tr><tr><td>Code-Review</td><td>:warning: 0</td><td>Found 2/30 approved changesets -- score normalized to 0</td></tr><tr><td>Dangerous-Workflow</td><td>:warning: -1</td><td>no workflows found</td></tr><tr><td>Maintained</td><td>:warning: 0</td><td>project is archived</td></tr><tr><td>Token-Permissions</td><td>:warning: -1</td><td>No tokens found</td></tr><tr><td>CII-Best-Practices</td><td>:warning: 0</td><td>no effort to earn an OpenSSF best practices badge detected</td></tr><tr><td>Security-Policy</td><td>:warning: 0</td><td>security policy file not detected</td></tr><tr><td>License</td><td>:warning: 0</td><td>license file not detected</td></tr><tr><td>Binary-Artifacts</td><td>:green_circle: 10</td><td>no binaries found in the repo</td></tr><tr><td>Signed-Releases</td><td>:warning: -1</td><td>no releases found</td></tr><tr><td>Pinned-Dependencies</td><td>:warning: -1</td><td>no dependencies found</td></tr><tr><td>Fuzzing</td><td>:warning: 0</td><td>project is not fuzzed</td></tr><tr><td>Branch-Protection</td><td>:warning: 0</td><td>branch protection not enabled on development/release branches</td></tr><tr><td>SAST</td><td>:warning: 0</td><td>SAST tool is not run on all commits -- score normalized to 0</td></tr><tr><td>Vulnerabilities</td><td>:green_circle: 9</td><td>1 existing vulnerabilities detected</td></tr></table></details></td></tr>
<tr><td><a href=https://github.com/jclouds/jclouds> maven/org.apache.jclouds:jclouds-core </a></td><td>2.7.3</td>
      <td>:warning: 2.5</td><td><details><summary>Details</summary><table><tr><th>Check</th><th>Score</th><th>Reason</th></tr><tr><td>Packaging</td><td>:warning: -1</td><td>packaging workflow not detected</td></tr><tr><td>Code-Review</td><td>:warning: 0</td><td>Found 2/30 approved changesets -- score normalized to 0</td></tr><tr><td>Dangerous-Workflow</td><td>:warning: -1</td><td>no workflows found</td></tr><tr><td>Maintained</td><td>:warning: 0</td><td>project is archived</td></tr><tr><td>Token-Permissions</td><td>:warning: -1</td><td>No tokens found</td></tr><tr><td>CII-Best-Practices</td><td>:warning: 0</td><td>no effort to earn an OpenSSF best practices badge detected</td></tr><tr><td>Security-Policy</td><td>:warning: 0</td><td>security policy file not detected</td></tr><tr><td>License</td><td>:warning: 0</td><td>license file not detected</td></tr><tr><td>Binary-Artifacts</td><td>:green_circle: 10</td><td>no binaries found in the repo</td></tr><tr><td>Signed-Releases</td><td>:warning: -1</td><td>no releases found</td></tr><tr><td>Pinned-Dependencies</td><td>:warning: -1</td><td>no dependencies found</td></tr><tr><td>Fuzzing</td><td>:warning: 0</td><td>project is not fuzzed</td></tr><tr><td>Branch-Protection</td><td>:warning: 0</td><td>branch protection not enabled on development/release branches</td></tr><tr><td>SAST</td><td>:warning: 0</td><td>SAST tool is not run on all commits -- score normalized to 0</td></tr><tr><td>Vulnerabilities</td><td>:green_circle: 9</td><td>1 existing vulnerabilities detected</td></tr></table></details></td></tr>
<tr><td> maven/org.checkerframework:checker-qual </td><td>3.43.0</td>
      <td>:green_circle: 3.5</td><td><details><summary>Details</summary><table><tr><th>Check</th><th>Score</th><th>Reason</th></tr><tr><td>Packaging</td><td>:warning: -1</td><td>packaging workflow not detected</td></tr><tr><td>Code-Review</td><td>:green_circle: 3</td><td>Found 10/27 approved changesets -- score normalized to 3</td></tr><tr><td>Maintained</td><td>:green_circle: 10</td><td>30 commit(s) and 6 issue activity found in the last 90 days -- score normalized to 10</td></tr><tr><td>Dangerous-Workflow</td><td>:warning: -1</td><td>no workflows found</td></tr><tr><td>Token-Permissions</td><td>:warning: -1</td><td>No tokens found</td></tr><tr><td>CII-Best-Practices</td><td>:warning: 0</td><td>no effort to earn an OpenSSF best practices badge detected</td></tr><tr><td>Security-Policy</td><td>:warning: 0</td><td>security policy file not detected</td></tr><tr><td>License</td><td>:green_circle: 9</td><td>license file detected</td></tr><tr><td>Branch-Protection</td><td>:warning: -1</td><td>internal error: error during branchesHandler.setup: internal error: some github tokens can't read classic branch protection rules: https://github.com/ossf/scorecard-action/blob/main/docs/authentication/fine-grained-auth-token.md</td></tr><tr><td>Signed-Releases</td><td>:warning: 0</td><td>Project has not signed or included provenance with any releases.</td></tr><tr><td>Binary-Artifacts</td><td>:warning: -1</td><td>internal error: failure checking for Gradle wrapper validating Action: failure listing workflow runs: internal error: ListWorkflowRunsByFileName: GET https://api.github.com/repos/typetools/checker-framework/actions/workflows/gradle-wrapper-validation.yml-DISABLED/runs?status=success: 404 Not Found []</td></tr><tr><td>Fuzzing</td><td>:warning: 0</td><td>project is not fuzzed</td></tr><tr><td>Vulnerabilities</td><td>:green_circle: 10</td><td>0 existing vulnerabilities detected</td></tr><tr><td>SAST</td><td>:warning: 0</td><td>SAST tool is not run on all commits -- score normalized to 0</td></tr><tr><td>Pinned-Dependencies</td><td>:warning: 0</td><td>dependency not pinned by hash detected -- score normalized to 0</td></tr></table></details></td></tr>
<tr><td><a href=https://github.com/eclipse-ee4j/angus-activation> maven/org.eclipse.angus:angus-activation </a></td><td>2.0.2</td>
      <td>:green_circle: 3.7</td><td><details><summary>Details</summary><table><tr><th>Check</th><th>Score</th><th>Reason</th></tr><tr><td>Dangerous-Workflow</td><td>:green_circle: 10</td><td>no dangerous workflow patterns detected</td></tr><tr><td>Packaging</td><td>:warning: -1</td><td>packaging workflow not detected</td></tr><tr><td>Maintained</td><td>:green_circle: 4</td><td>5 commit(s) and 0 issue activity found in the last 90 days -- score normalized to 4</td></tr><tr><td>Binary-Artifacts</td><td>:green_circle: 10</td><td>no binaries found in the repo</td></tr><tr><td>Code-Review</td><td>:warning: 2</td><td>Found 5/22 approved changesets -- score normalized to 2</td></tr><tr><td>Token-Permissions</td><td>:warning: 0</td><td>detected GitHub workflow tokens with excessive permissions</td></tr><tr><td>Pinned-Dependencies</td><td>:warning: 0</td><td>dependency not pinned by hash detected -- score normalized to 0</td></tr><tr><td>CII-Best-Practices</td><td>:warning: 0</td><td>no effort to earn an OpenSSF best practices badge detected</td></tr><tr><td>Fuzzing</td><td>:warning: 0</td><td>project is not fuzzed</td></tr><tr><td>License</td><td>:green_circle: 10</td><td>license file detected</td></tr><tr><td>Branch-Protection</td><td>:warning: 0</td><td>branch protection not enabled on development/release branches</td></tr><tr><td>Signed-Releases</td><td>:warning: -1</td><td>no releases found</td></tr><tr><td>Security-Policy</td><td>:green_circle: 10</td><td>security policy file detected</td></tr><tr><td>SAST</td><td>:warning: 0</td><td>SAST tool is not run on all commits -- score normalized to 0</td></tr><tr><td>Vulnerabilities</td><td>:warning: 0</td><td>22 existing vulnerabilities detected</td></tr></table></details></td></tr>
<tr><td> maven/org.glassfish.jaxb:jaxb-core </td><td>4.0.5</td>
      <td>:green_circle: 6.6</td><td><details><summary>Details</summary><table><tr><th>Check</th><th>Score</th><th>Reason</th></tr><tr><td>Code-Review</td><td>:green_circle: 6</td><td>Found 17/25 approved changesets -- score normalized to 6</td></tr><tr><td>Maintained</td><td>:green_circle: 10</td><td>13 commit(s) and 2 issue activity found in the last 90 days -- score normalized to 10</td></tr><tr><td>Packaging</td><td>:warning: -1</td><td>packaging workflow not detected</td></tr><tr><td>Dangerous-Workflow</td><td>:green_circle: 10</td><td>no dangerous workflow patterns detected</td></tr><tr><td>Token-Permissions</td><td>:warning: 0</td><td>detected GitHub workflow tokens with excessive permissions</td></tr><tr><td>CII-Best-Practices</td><td>:warning: 0</td><td>no effort to earn an OpenSSF best practices badge detected</td></tr><tr><td>License</td><td>:green_circle: 10</td><td>license file detected</td></tr><tr><td>Binary-Artifacts</td><td>:green_circle: 10</td><td>no binaries found in the repo</td></tr><tr><td>Pinned-Dependencies</td><td>:warning: 0</td><td>dependency not pinned by hash detected -- score normalized to 0</td></tr><tr><td>Branch-Protection</td><td>:warning: 0</td><td>branch protection not enabled on development/release branches</td></tr><tr><td>Security-Policy</td><td>:green_circle: 10</td><td>security policy file detected</td></tr><tr><td>Fuzzing</td><td>:green_circle: 10</td><td>project is fuzzed</td></tr><tr><td>Signed-Releases</td><td>:warning: -1</td><td>no releases found</td></tr><tr><td>SAST</td><td>:green_circle: 7</td><td>SAST tool detected but not run on all commits</td></tr><tr><td>Vulnerabilities</td><td>:green_circle: 10</td><td>0 existing vulnerabilities detected</td></tr></table></details></td></tr>
<tr><td> maven/org.glassfish.jaxb:jaxb-runtime </td><td>4.0.5</td>
      <td>:green_circle: 6.6</td><td><details><summary>Details</summary><table><tr><th>Check</th><th>Score</th><th>Reason</th></tr><tr><td>Code-Review</td><td>:green_circle: 6</td><td>Found 17/25 approved changesets -- score normalized to 6</td></tr><tr><td>Maintained</td><td>:green_circle: 10</td><td>13 commit(s) and 2 issue activity found in the last 90 days -- score normalized to 10</td></tr><tr><td>Packaging</td><td>:warning: -1</td><td>packaging workflow not detected</td></tr><tr><td>Dangerous-Workflow</td><td>:green_circle: 10</td><td>no dangerous workflow patterns detected</td></tr><tr><td>Token-Permissions</td><td>:warning: 0</td><td>detected GitHub workflow tokens with excessive permissions</td></tr><tr><td>CII-Best-Practices</td><td>:warning: 0</td><td>no effort to earn an OpenSSF best practices badge detected</td></tr><tr><td>License</td><td>:green_circle: 10</td><td>license file detected</td></tr><tr><td>Binary-Artifacts</td><td>:green_circle: 10</td><td>no binaries found in the repo</td></tr><tr><td>Pinned-Dependencies</td><td>:warning: 0</td><td>dependency not pinned by hash detected -- score normalized to 0</td></tr><tr><td>Branch-Protection</td><td>:warning: 0</td><td>branch protection not enabled on development/release branches</td></tr><tr><td>Security-Policy</td><td>:green_circle: 10</td><td>security policy file detected</td></tr><tr><td>Fuzzing</td><td>:green_circle: 10</td><td>project is fuzzed</td></tr><tr><td>Signed-Releases</td><td>:warning: -1</td><td>no releases found</td></tr><tr><td>SAST</td><td>:green_circle: 7</td><td>SAST tool detected but not run on all commits</td></tr><tr><td>Vulnerabilities</td><td>:green_circle: 10</td><td>0 existing vulnerabilities detected</td></tr></table></details></td></tr>
<tr><td> maven/org.glassfish.jaxb:txw2 </td><td>4.0.5</td>
      <td>:green_circle: 6.6</td><td><details><summary>Details</summary><table><tr><th>Check</th><th>Score</th><th>Reason</th></tr><tr><td>Code-Review</td><td>:green_circle: 6</td><td>Found 17/25 approved changesets -- score normalized to 6</td></tr><tr><td>Maintained</td><td>:green_circle: 10</td><td>13 commit(s) and 2 issue activity found in the last 90 days -- score normalized to 10</td></tr><tr><td>Packaging</td><td>:warning: -1</td><td>packaging workflow not detected</td></tr><tr><td>Dangerous-Workflow</td><td>:green_circle: 10</td><td>no dangerous workflow patterns detected</td></tr><tr><td>Token-Permissions</td><td>:warning: 0</td><td>detected GitHub workflow tokens with excessive permissions</td></tr><tr><td>CII-Best-Practices</td><td>:warning: 0</td><td>no effort to earn an OpenSSF best practices badge detected</td></tr><tr><td>License</td><td>:green_circle: 10</td><td>license file detected</td></tr><tr><td>Binary-Artifacts</td><td>:green_circle: 10</td><td>no binaries found in the repo</td></tr><tr><td>Pinned-Dependencies</td><td>:warning: 0</td><td>dependency not pinned by hash detected -- score normalized to 0</td></tr><tr><td>Branch-Protection</td><td>:warning: 0</td><td>branch protection not enabled on development/release branches</td></tr><tr><td>Security-Policy</td><td>:green_circle: 10</td><td>security policy file detected</td></tr><tr><td>Fuzzing</td><td>:green_circle: 10</td><td>project is fuzzed</td></tr><tr><td>Signed-Releases</td><td>:warning: -1</td><td>no releases found</td></tr><tr><td>SAST</td><td>:green_circle: 7</td><td>SAST tool detected but not run on all commits</td></tr><tr><td>Vulnerabilities</td><td>:green_circle: 10</td><td>0 existing vulnerabilities detected</td></tr></table></details></td></tr>
</table></details><h2>Scanned Files</h2>
<ul><li>settings.gradle</li></ul>
